systemtests/src/main/java/io/enmasse/systemtest/cmdclients/KubeCMDClient.java

```diff
@@ -290,9 +291,24 @@ public static ExecutionResultData applyFromFile(String namespace, Path path) {
         return execute(Arrays.asList(CMD, "-n", namespace, "apply", "-f", path.toString()), DEFAULT_SYNC_TIMEOUT, true);
     }
 
+    public static ExecutionResultData deleteFromFile(String namespace, Path path) {
+        Objects.requireNonNull(namespace);
+        Objects.requireNonNull(path);
+        return execute(Arrays.asList(CMD, "-n", namespace, "delete", "-f", path.toString()), DEFAULT_SYNC_TIMEOUT, true);
+    }
+
     public static String getMessagingEndpoint(String namespace, String addressspace) {
         Objects.requireNonNull(namespace);
         Objects.requireNonNull(addressspace);
         return execute(Arrays.asList(CMD, "-n", namespace, "get", "addressspace", addressspace, "-o", "jsonpath={.status.endpointStatuses[?(@.name==\"messaging\")].externalHost}"), DEFAULT_SYNC_TIMEOUT, true, false).getStdOut();
     }
+
+    public static ExecutionResultData labelResource(String resource, String resourceName, String key, String value) {
```

Comment:
why do you label resources using executor, it isn't possible via fabric8 client?

Reply:
changed